docs/source/cli.rst

```diff
@@ -0,0 +1,72 @@
+.. highlight:: console
+
+.. role:: bash(code)
+
+.. _cli:
+
+Command Line Interface
+======================
+
+When git-machete is installed it is added to your path, so it can be called from command line: :bash:`git-machete <command> <options>`. git-machete comes with a wide variety of commands and their options and in this site you can find their documentation.
```

Comment:
```suggestion
When git-machete is installed, it adds `machete` command to `git`, so it can be called from command line: :bash:`git machete <command> <options>`. git-machete comes with a wide variety of commands and their options and in this site you can find their documentation.
```